open_src/github.com/VictoriaMetrics/VictoriaMetrics/lib/fs/fs_windows.go

```diff
@@ -0,0 +1,31 @@
+package fs
+
+import (
+	"unsafe"
+
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
```

Comment:
Do not use this `logger`. Use "github.com/openGemini/openGemini/lib/logger" instead.